.travis.yml

```diff
@@ -1,6 +1,12 @@
+dist: trusty
 language: node_js
-
 node_js:
   - 7
```

Comment:
I'm not sure that it's necessary, because we don't use node-specific features, and transpiled code should work with node from v4.

But if there are some issues with, we can run tests on all versions of course